pages/about/using-wai-material.md

```diff
@@ -8,7 +8,7 @@ permalink: /about/using-wai-material/
 ref: /about/using-wai-material/
 
 footer: >
-  <p><strong>Date:</strong> Updated 1 February 2023.</p>
+  <p><strong>Date:</strong> Updated @@ Month 2024.</p>
   <p><strong>Editor:</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a>.</p>
   <p>Developed with the <a href="http://www.w3.org/People/domain?domain=Web+Accessibility+Initiative">WAI staff</a> and <a href="https://www.w3.org/People/functions/comm">W3C Communications staff</a>.</p>
```

Comment:
In parallel, please ask w3t-sys to create a WAI Team function since we killed domains the link for WAI no longer works. 
The link for Comm is <https://www.w3.org/staff/comm/>

Reply:
Meanwhile, https://www.w3.org/staff/wai/ now exists.
I don't know which is more convenient between editing each occurrence of "http(s)://www.w3.org/People/domain?domain=Web+Accessibility+Initiative" or asking the Systems Team if redirects can be put in place.